drivers/dma/dma_mcux_lpc.c

```diff
@@ -492,6 +498,9 @@ static int dma_mcux_lpc_configure(const struct device *dev, uint32_t channel,
 				dst_inc,
 				block_config->block_size);
 	}
+	/* DMA controller requires that the address be aligned to transfer size */
```

Comment:
I am not aware of such a alignment requirement. 
There is a requirement for the DMA descriptors which this driver addresses.

Reply:
In the RT500 Reference Manual, section 16.4.2.4 says:

> 16.4.2.4 Address alignment for data transfers
> Transfers of 16 bit width require an address alignment to a multiple of 2 bytes. Transfers of 32 bit width require an address alignment to a multiple of 4 bytes. Transfers of 8 bit width can be at any address